quint/package.json

```diff
@@ -50,18 +50,21 @@
   "dependencies": {
     "@grpc/grpc-js": "^1.8.14",
     "@grpc/proto-loader": "^0.7.7",
+    "@octokit/request": "^8.1.1",
     "@sweet-monads/either": "^3.0.1",
     "@sweet-monads/maybe": "^3.1.0",
     "@types/line-column": "^1.0.0",
     "@types/seedrandom": "^3.0.4",
     "antlr4ts": "^0.5.0-alpha.4",
     "chalk": "^4.1.2",
     "eol": "^0.9.1",
+    "gunzip-maybe": "^1.4.2",
```

Comment:
Still needed?

Reply:
Oof, thanks for the careful review! I'm really bad at tracking dependencies 😅
d2045ae9